File: dvd-store/src/app/core/models/Customer.ts
import { Address } from "./Address";
import { Store } from "./Store";

export interface Customer {
    customerId: number;
    firstName: string;
    lastName: string;
    email: string;
    activebool: boolean;
    createDate: string;
    lastUpdate: string;
    active: number;
    storeId: number;
    store: Store;
    addressId: number;
    address: Address;
}

File: backend/src/Backend/DvdStore.Domain/Entities/Customer.cs
namespace DvdStore.Domain.Entities;

public class Customer
{
    public int CustomerId { get; set; }
    public string ?FirstName { get; set; }
    public string ?LastName { get; set; }
    public string ?Email { get; set; }
    public bool Activebool { get; set; } 
    public string CreateDate { get; set; } = string.Empty;
    public string LastUpdate { get; set; } = string.Empty;
    public int Active { get; set; }
    public int StoreId { get; set; }
    public Store Store { get; set; } = null!;
    public int AddressId { get; set; }
    public Address Address { get; set; } = null!;
}
